### PR TITLE
update icewind/streams to 0.4.1

### PR DESCRIPTION
**Pull request by icewind1991**
Contains a fix for https://github.com/owncloud/core/issues/23781

### CODE REVIEW

**Comment by PVince81** [comment]
9.2

**Comment by PVince81** [comment]
As discussed in the core ticket, acceptable for 9.1 :+1:

File: composer/autoload_classmap.php [hunk 1]
```diff
@@ -721,6 +721,19 @@
...
+    'Icewind\\Streams\\Tests\\DirectoryWrapperDummy' => $vendorDir . '/icewind/streams/tests/DirectoryWrapper.php',
+    'Icewind\\Streams\\Tests\\DirectoryWrapperNull' => $vendorDir . '/icewind/streams/tests/DirectoryWrapper.php',
+    'Icewind\\Streams\\Tests\\FailWrapper' => $vendorDir . '/icewind/streams/tests/RetryWrapper.php',
+    'Icewind\\Streams\\Tests\\IteratorDirectory' => $vendorDir . '/icewind/streams/tests/IteratorDirectory.php',
+    'Icewind\\Streams\\Tests\\NullWrapper' => $vendorDir . '/icewind/streams/tests/NullWrapper.php',
+    'Icewind\\Streams\\Tests\\PartialWrapper' => $vendorDir . '/icewind/streams/tests/RetryWrapper.php',
+    'Icewind\\Streams\\Tests\\RetryWrapper' => $vendorDir . '/icewind/streams/tests/RetryWrapper.php',
+    'Icewind\\Streams\\Tests\\SeekableWrapper' => $vendorDir . '/icewind/streams/tests/SeekableWrapper.php',
+    'Icewind\\Streams\\Tests\\UrlCallBack' => $vendorDir . '/icewind/streams/tests/UrlCallBack.php',
+    'Icewind\\Streams\\Tests\\Wrapper' => $vendorDir . '/icewind/streams/tests/Wrapper.php',
```
Comment (nickvergessen) [inline composer/autoload_classmap.php]: This looks wrong

Comment (rullzer) [inline composer/autoload_classmap.php]: yep remove the test folder and rerun `composer dumpautoload -o -a`

File: composer/autoload_classmap.php [hunk 2]
```diff
@@ -1488,6 +1501,15 @@
...
+    'Symfony\\Component\\Console\\Tests\\Helper\\LegacyProgressHelperTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Helper/LegacyProgressHelperTest.php',
+    'Symfony\\Component\\Console\\Tests\\Helper\\LegacyTableHelperTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Helper/LegacyTableHelperTest.php',
+    'Symfony\\Component\\Console\\Tests\\Helper\\ProcessHelperTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Helper/ProcessHelperTest.php',
+    'Symfony\\Component\\Console\\Tests\\Helper\\ProgressBarTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php',
+    'Symfony\\Component\\Console\\Tests\\Helper\\QuestionHelperTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php',
+    'Symfony\\Component\\Console\\Tests\\Helper\\TableTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Helper/TableTest.php',
+    'Symfony\\Component\\Console\\Tests\\Logger\\ConsoleLoggerTest' => $vendorDir . '/symfony/console/Symfony/Component/Console/Tests/Logger/ConsoleLoggerTest.php',
```
Comment (rullzer) [inline composer/autoload_classmap.php]: Same here

**Comment by PVince81** [comment]
@icewind1991 please adjust

**Comment by icewind1991** [comment]
Done

**Comment by PVince81** [comment]
Still works :+1:

File: composer/installed.json [hunk 3]
```diff
@@ -43,7 +43,8 @@
             "common",
             "event",
             "exception"
-        ]
+        ],
+        "abandoned": "guzzle/guzzle"
```
Comment (PVince81) [inline composer/installed.json]: not sure why, but I guess composer generated this

Comment (nickvergessen) [inline composer/installed.json]: yes

**Comment by nickvergessen** [comment]
:+1:

**Pull request merged**